flixel/animation/FlxAnimation.hx

```diff
@@ -49,6 +49,16 @@ class FlxAnimation extends FlxBaseAnimation
 	public var reversed(default, null):Bool = false;
 	
 	/**
+	 * Whether or not the frames of this animation are horizontally flipped
+	 */
+	public var flipX(default, null):Bool = false;
```

Comment:
Considering these are checked during each render, is there a reason to make them read-only? Setting them after creation should work.